1-server-side/msal-web-api-sample/msal-obo-sample/pom.xml

```diff
@@ -29,7 +29,7 @@
         <dependency>
             <groupId>com.microsoft.graph</groupId>
             <artifactId>microsoft-graph</artifactId>
-            <version>6.5.1</version>
```

Comment:
Why did you have to downgrade?

Reply:
Similar to the nimbus/JSON dependency, this was upgraded in my last PR and it had undiscovered API changes. However, this was not giving any dependabot alerts so I downgraded to the previous major version rather than do additional refactoring for something that's not the focus of the sample (it's still newer than what we were using before, just not the latest).